proposals/0015-extended-command-info.md

```diff
@@ -0,0 +1,226 @@
+# Extended Command Information
+
+* Proposal: [0015](0015-extended-command-info.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
```

Comment:
Is "Under Consideration" the correct status for this?

Reply:
It's what the others are all set to. I intend to change them all in a followup